src/Controller/Api/DatasetSubmissionController.php

```diff
@@ -352,6 +353,45 @@ public function getUploadedFilesAction(int $id)
         return array($fileInfo);
     }
 
+    /**
```

Comment:
This whole block isn't anything new, just moved it here. Because it uses DatasetSubmission Entity to get all the related files of a Fileset